src/templates/options.js

```diff
@@ -19,3 +19,9 @@ for (const [key, value] of Object.entries(options)) {
     }
 }
 %>
+
+/* <% if (options.langDir) { %> */
+export const ASYNC_LOCALES = {
+  <%= options.locales.map(l => `'${l.code}': () => import('${options.langDir}/${l.file || l.iso}' /* webpackChunkName: "lang-${l.code}" */)`).join(',\n  ') %>
```

Comment:
The `webpackChunkName` could be `lang-${l.file}`. That way if multiple locales reuse the same language file, those wouldn't create separate chunks.